python/ray/util/sgd/v2/examples/transformers/README.rst

```diff
@@ -0,0 +1,53 @@
+HuggingFace Transformers Glue Fine-tuning Example
+=================================================
+
+We've ported the ``huggingface/transformers/examples/pytorch/text-classification/run_glue_no_trainer.py`` example to
+RaySGD. This example enables fine-tuning the library models for sequence classification on the GLUE benchmark: General Language Understanding Evaluation.
+
+This script can fine-tune the following models:
+ CoLA, SST-2, MRPC, STS-B, QQP, MNLI, QNLI, RTE, WNLI.
+
+Additional information can be found at the `HuggingFace Repository
+<https://github.com/huggingface/transformers/tree/master/examples/pytorch/text-classification>`_.
+
+Single-node CPU training
+----------------------------
+
+To run an example tuning MRPC locally using CPU:
+
+.. code-block:: bash
+
+    export TASK_NAME=mrpc
+
+    ray.init()
```

Comment:
ray.init() shouldn't go here right?

Reply:
Yeah good point, updated this as well as the code so that if no Ray configs are passed in then it goes through the same code path as [run_glue_no_trainer.py](https://github.com/huggingface/transformers/blob/master/examples/pytorch/text-classification/run_glue_no_trainer.py).